src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -1,47 +1,163 @@
 package core.basesyntax;
 
 import java.util.List;
+import java.util.Objects;
```

Comment:
Avoid using the `Objects` class as per the checklist: 'Don't use class Objects.'